### ISSUE TITLE
It says "Fuck It" at times.

### ISSUE THREAD

**Issue by AnimishY** [Asher-Ul-Haque/Calculator#1]

**Comment by Asher-Ul-Haque** [comment]
Yo @AnimishY, bro, be reviewing my code. 
So, there literally is a function called Fuck it inside it. I never found it working while testing, though. Thanks for pointing out.
><b><H1>Why function Fuck it exists:</H1></b> 
>><i><H2>How user input works</H2></i>
The user can switch on or off the decimal button.
This means that when the decimal button is pressed <b>Off</b> then any number pressed will be <b>appended</b> (and not added) to the number on the display <b> before the decimal point </b>
When the decimal button is pressed <b>On</b> then any number pressed will be <b>appended</b> (and not added) to the number on the display <b> after the decimal point </b>

>> <i> <H2>The error </H2></i>
Sometimes, I didn't know why, at the time, the backspace button would not remove the numbers after the decimal place.
In frustration, I made this function to bully my code into working. The code definition is nothing special; it is just the same code it was supposed to work with. Except with one line to print('Fuck It').
Because it never printed FuckIt for me. I thought I had successfully bullied my code.
I hope there is no error in the GUI. The fuck should be printed only in the terminal.

>> <i> <H2>The fix </H2></i>
remove line 134 that prints Fuck it.
On second thought, remove all print lines
